deployments/arbitrum/usdc.n/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC(native)",
+  "symbol": "cUSDCNv3",
```

Comment:
I think we should just use `cUSDCv3`. Could be slighlty confusing to have two `cUSDCv3`'s, but we can potentially upgrade the old one to be `cUSDC.ev3`. Though I don't think that's high priority.

Reply:
That make sense, I can go ahead to to rename `usdc` -> `usdc.e` and rename this native from `usdc.n` -> `usdc`